### DISCUSSION TITLE
FREE Galaxy brain

### DISCUSSION THREAD

**Discussion by MitkoVtori** [drknzz/GitHub-Achievements#50] [Q&A] [answered]
In the comments give me link to your question, I'll comment, make my coment set as answer. THen I'll make your comment answer :)

**Comment by dlmalhao** [comment, marked as answer]
Hey !! Here you have it :)   
https://github.com/drknzz/GitHub-Achievements/discussions/52

**Comment by Kavitate** [comment]
This still work?

**Reply by MitkoVtori** [reply]
answer this #59
I'll set you as answer :)
